docroot/profiles/custom/cgov_site/modules/custom/cgov_redirect_manager/cgov_redirect_manager.module

```diff
@@ -37,6 +40,7 @@ function cgov_redirect_manager_cron() {
   // Process all cgov_redirect_manager_queue_worker in the given time.
   while ((!$time_limit || time() < $end) && ($item = $queue->claimItem())) {
     try {
+      /* @phpstan-ignore-next-line */
```

Comment:
Can you set the type hint for $queue_worker so you don't need the ignore?

You should be able to use `\Drupal\Core\Queue\QueueWorkerBase`.

Reply:
Error is coming not on $queue_worker but coming on data. Cannot access property $data on object|true.